app/views/galleries/_list_section.haml

```diff
@@ -0,0 +1,31 @@
+%tr.gallery{id: "gallery#{gallery ? gallery.id : 0}"}
+  %td
+    = form_tag delete_multiple_icons_path, method: :delete do
+      %table.gallery-table
+        %tr
+          - klass = cycle('even', 'odd')
+          %th.subber{class: klass} Keyword
+          %th.subber{class: klass} Credit
+          %th.subber{class: klass} Times Used
+          %th.subber{class: klass} Posts In
+          %th.subber{class: klass}
+        - icons.pluck(:id, :keyword, :credit).each do |icon_id, keyword, credit|
+          - klass = cycle('even', 'odd')
+          %tr
+            %td{class: klass}
+              = link_to keyword, icon_path(id: icon_id)
+            %td{class: klass}= credit
+            %td{class: klass}= @times_used[icon_id]
+            %td{class: klass}= @posts_used[icon_id]
+
+            %td.width-70.right-align{class: klass}
+              - if is_owner && !skip_forms
+                = check_box_tag :"marked_ids[]", icon_id
```

Comment:
Should this be wrapped in `.select-button`? Also should it have `, false, id: nil` on the end? `_icon_view.haml` seems to do that, and I'm not sure if they're necessary.

Reply:
...yes it should have the false id:nil. not sure about the .select-button